src/hooks/useLocale.jsx

```diff
@@ -0,0 +1,23 @@
+// import { useState, useEffect } from "react";
+import Spanish from "../lang/es-US.json";
+import English from "../lang/en-US.json";
+
+// detect the language preference of the user's browser
+const useLocale = () => {
+  const locale = window.navigator.userLanguage || window.navigator.language;
+  // const [messages, setMessages] = useState(English);
+
+  // useEffect(() => {
+  let messages;
+  if (locale === "en-US") {
+    messages = English;
+    // setMessages(English);
+  } else {
+    messages = Spanish;
+    // setMessages(Spanish);
+  }
```

Comment:
💡 Could you make this a switch statement. That should be easier to maintain when we add more languages.

Reply:
Addressed:  6447f3c407ffb09cf1bec564ec331c363a951e9f